### DISCUSSION TITLE
Podman upgrade

### DISCUSSION THREAD

**Discussion by moinkha3** [containers/podman#24052] [General]
i have podman of v4.4.1, i need it to be upgraded to the latest stable version.
Please help

**Comment by moinkha3** [comment]
Can someone please help me on this

**Comment by moinkha3** [comment]
@rhatdan @Luap99

**Comment by rhatdan** [comment]
What Distribution?

Podman upstream is not responsible for providing software for particular distributions, other then for Mac and Windows.

Some of the maintainers who work for Red Hat also work on the Fedora/Centos Stream and RHEL distributions and work to keep these distributions up 2 date.  OpenSuSe does a good job of keeping up, Debian is updated via contributor who does a good job of keeping up. Ubuntu not so much.

**Comment by moinkha3** [comment]
RHEL 8.x
Also, podman is not managing the containers properly.
containers are assigned and not being taken off when not in use by the podman.

Due to this there is too much load and system crashes.
